From cf5ba66a27eb52cc5d59b75610f8af73adea99a8 Mon Sep 17 00:00:00 2001
From: Song Bing <bing.song@nxp.com>
Date: Tue, 10 Jul 2018 17:36:06 -0700
Subject: [PATCH] MLK-18834 ion: Enable ion on Kernel 4.14

commit acb13077634fe6df4f863aa8bf66c6bccf2f80b5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable ion driver on Linux platform for Kernel 4.14.

Signed-off-by: Song Bing <bing.song@nxp.com>
[Arul: Fix merge conflicts]
Signed-off-by: Arulpandiyan Vadivel <arulpandiyan_vadivel@mentor.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/staging/Makefile        | 2 +-
 drivers/staging/android/Kconfig | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index 0da0d3f0b5e4..6fb200b539a2 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -22,7 +22,7 @@ obj-$(CONFIG_FB_SM750)		+= sm750fb/
 obj-$(CONFIG_USB_EMXX)		+= emxx_udc/
 obj-$(CONFIG_SPEAKUP)		+= speakup/
 obj-$(CONFIG_MFD_NVEC)		+= nvec/
-obj-$(CONFIG_ANDROID)		+= android/
+obj-y		+= android/
 obj-$(CONFIG_STAGING_BOARD)	+= board/
 obj-$(CONFIG_LTE_GDM724X)	+= gdm724x/
 obj-$(CONFIG_FIREWIRE_SERIAL)	+= fwserial/
diff --git a/drivers/staging/android/Kconfig b/drivers/staging/android/Kconfig
index d6d605d5cbde..784925f5ab1f 100644
--- a/drivers/staging/android/Kconfig
+++ b/drivers/staging/android/Kconfig
@@ -26,4 +26,6 @@ source "drivers/staging/android/ion/Kconfig"
 
 endif # if ANDROID
 
+source "drivers/staging/android/ion/Kconfig"
+
 endmenu
-- 
2.17.1

